Code/Tools/ProjectManager/Platform/Linux/ProjectUtils_linux.cpp

```diff
@@ -6,15 +6,46 @@
  */
 
 #include <ProjectUtils.h>
+#include <ProjectManagerDefs.h>
+#include <QProcessEnvironment>
 
 namespace O3DE::ProjectManager
 {
     namespace ProjectUtils
     {
-        AZ::Outcome<void, QString> FindSupportedCompilerForPlatform()
+        AZ::Outcome<QProcessEnvironment, QString> GetCommandLineProcessEnvironment()
         {
-            // Compiler detection not supported on platform
-            return AZ::Success();
+            return AZ::Success(QProcessEnvironment(QProcessEnvironment::systemEnvironment()));
+        }
+
+        AZ::Outcome<QString, QString> FindSupportedCompilerForPlatform()
+        {
+            // Validate that cmake is installed and is in the command line
+            auto whichCMakeResult = ProjectUtils::ExecuteCommandResult("which", QStringList{ProjectCMakeCommand}, QProcessEnvironment::systemEnvironment());
+            if (!whichCMakeResult.IsSuccess())
+            {
+                return AZ::Failure(QObject::tr("CMake not found. \n\n"
+                    "Make sure that the minimum version of CMake is installed and available from the command prompt. "
+                    "Refer to the <a href='https://o3de.org/docs/welcome-guide/setup/requirements/#cmake'>O3DE requirements</a> page for more information."));
+            }
+
+            // Look for the first compatible version of clang. The list below will contain the known clang compilers that have been tested for O3DE.
+            QStringList supportedClangCommands = {"clang-12|clang++-12", "clang-6|clang++-6"};
```

Comment:
I personally think this variable should be exposed somewhere a little more visible like ProjectManagerDefs.h possibly since it is really important and will need to be updated.

Reply:
Moving it out to ProjectManagerDefs.h will go against our PAL philosophy. I'll move it as a static on the top of the file with comments instead, since adding an additional include is unnecessary for this.

Reply:
Oh I was wondering about the PAL side of that, ok that works for me!